src/xcode/ENA/ENA/Source/Scenes/TicketValidation/CertificateSelection/TicketValidationCertificateSelectionViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class TicketValidationCertificateSelectionViewController: DynamicTableViewController {
```

Comment:
This view controller needs to conform to the `DismissHandling` protocol so the alert is shown when the user pulls down as well. I already changed it for the first consent in my [new PR](https://github.com/corona-warn-app/cwa-app-ios/pull/3878/files#diff-d27a52c67d3c582fffc93330f1084402c8320d6371e548d745d939882580d746). And there is a distinction missing in you implementation: If there are no matching certificates, the alert should not be shown and we should call `cancel()` on the `ticketValidation` and the screen should just be closed.